doc/classes/TextServer.xml

```diff
@@ -53,6 +59,122 @@
+			<param index="0" name="list" type="RID" />
+			<param index="1" name="layer" type="int" />
+			<param index="2" name="transform" type="Transform2D" />
+			<param index="3" name="callback" type="Callable" />
+			<description>
+				Adds custom draw call to a draw list [param list]. [param callback] is called with one argument, [RID] of canvas item to draw it to.
+			</description>
+		</method>
+		<method name="draw_list_add_hexbox" qualifiers="const">
+			<return type="void" />
+			<param index="0" name="list" type="RID" />
+			<param index="1" name="layer" type="int" />
+			<param index="2" name="transform" type="Transform2D" />
+			<param index="3" name="pos" type="Vector2" />
+			<param index="4" name="index" type="int" />
+			<param index="5" name="size" type="int" />
+			<param index="6" name="modulate" type="Color" />
+			<description>
+				Adds draw call for box displaying character hexadecimal code to a draw list [param list].
+			</description>
+		</method>
+		<method name="draw_list_add_line" qualifiers="const">
+			<return type="void" />
+			<param index="0" name="list" type="RID" />
+			<param index="1" name="layer" type="int" />
+			<param index="2" name="transform" type="Transform2D" />
+			<param index="3" name="start" type="Vector2" />
+			<param index="4" name="end" type="Vector2" />
+			<param index="5" name="width" type="float" />
+			<param index="6" name="dash" type="float" />
+			<param index="7" name="modulate" type="Color" />
+			<description>
+				Adds draw call for a line to a draw list [param list].
+			</description>
+		</method>
+		<method name="draw_list_add_rect" qualifiers="const">
+			<return type="void" />
+			<param index="0" name="list" type="RID" />
+			<param index="1" name="layer" type="int" />
+			<param index="2" name="transform" type="Transform2D" />
+			<param index="3" name="rect" type="Rect2" />
+			<param index="4" name="filled" type="bool" />
+			<param index="5" name="modulate" type="Color" />
+			<description>
+				Adds draw call for a rectangle to a draw list [param list].
+			</description>
+		</method>
+		<method name="draw_list_add_texture" qualifiers="const">
+			<return type="void" />
+			<param index="0" name="list" type="RID" />
+			<param index="1" name="layer" type="int" />
+			<param index="2" name="transform" type="Transform2D" />
+			<param index="3" name="texture" type="RID" />
+			<param index="4" name="rect" type="Rect2" />
+			<param index="5" name="modulate" type="Color" />
+			<description>
+				Adds draw call for a texture to a draw list [param list].
+			</description>
+		</method>
+		<method name="draw_list_draw">
+			<return type="void" />
+			<param index="0" name="list" type="RID" />
+			<param index="1" name="ci" type="RID" />
+			<param index="2" name="free" type="bool" default="false" />
+			<description>
+				Draws all draw calls in the draw list to the canvas item [param ci]. If [param free] is [code]true[/code], draw list is automatically deleted.
```

Comment:
Lots of things in programming happen "automatically", but feel free to disregard if you disagree.
Also "draw" being repeated 3 times in quick succession is really, really nasty. How else can this operation be named?
```suggestion
				Processes all draw calls in the draw list [param list] to the canvas item [param ci]. If [param free] is [code]true[/code], the draw list is also deleted.
```